content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -847,9 +847,9 @@ is turned on.
   PVs of that default. Specifying a default StorageClass is done by setting the
   annotation `storageclass.kubernetes.io/is-default-class` equal to `true` in
   a StorageClass object. If the administrator does not specify a default, the
-  cluster responds to PVC creation as if the admission plugin were turned off. If
-  more than one default is specified, the admission plugin forbids the creation of
-  all PVCs.
+  cluster responds to PVC creation as if the admission plugin were turned off. If more than one
+  default StorageClass is specified, the newest default is used when the
+  PVC is dynamically provisioned.
```

Comment:
@RomanBednar @jsafrane Can you verify this?  Based on this PR https://github.com/kubernetes/kubernetes/pull/117064/files, it is not always the newest.

Reply:
IMO it will be very rare / impossible that two StorageClasses have the same CreationTimestamp. IMO the sentence is good enough for users.

Reply:
^^^ Agreed, chances to hit that condition is very low.